ci/build-test.cmd

```diff
@@ -0,0 +1,3 @@
+@echo off
+
+"c:\program files\git\bin\bash.exe" -c "./ci/build-test.sh"
```

Comment:
Minor nit: Can we move these into `ci/buildkite` or similar? As its effectively a pass through to the `build-test` script

Reply:
I'd prefer to keep it side by side, since it _also_ allows you to run your build from a Windows shell directory rather than assuming git-bash is the terminal of choice. What do you think?

Reply:
Hmm.. that is true, but the `cmd` file also assumes that you have git bash and assumes the location of the `exe` as well